tests/common.py

```diff
@@ -156,3 +159,35 @@ def raise_on_failure(response, custom_message=""):
             custom_message
             + f"Received failure response `{error_message}`. Whole response object: {response}"
         )
+
+
+def post_model_outputs_with_retries(
```

Comment:
The function name might be a little misleading since we don't retry in prod, not sure if it's worth naming cause that can get a bit verbose.